### PR TITLE
Transfer users via local storage

### LINKED ISSUE #47: Do not block or warn before blocking accounts I follow
> Und noch ein Vorschlag:
> - Rückfrage bei Accounts, denen man selbst folgt oder die einen folgen, vor einem Block. Also eine Art Whitelist (Einstellungen+Listen könnten ja in der Cloud-Speicherfunktion des Browsers hinterlegt werden).

https://twitter.com/Pertsch/status/1234152641472684036?s=20

**Alternative:**
Let the use maintain a list of accounts that not be blocked.

### LINKED ISSUE #88: Avoid blocklist division and pass whole blocklist at once to block service
When collecting very large lists of likers, it can be very annoying to click on all these buttons, open the block service in several tabs and run into errors when you are not logged in.

grafik

**ToDo:**

* [x] Store collected users in local storage
* [x] read them from local storage on the service side
* [x] divide API list in several API calls to avoid error response

### PR DESCRIPTION
**Pull request by dmstern**
fixes #88 
fixes #47

**ToDo:**

* https://github.com/pkreissel/ichbinhier_twittertools/issues/16

### CODE REVIEW

File: src/Storage.ts [hunk 1]
```diff
@@ -88,4 +91,15 @@ export default class Storage {
 			await client.storage.local.set({ [Key.packageVersion]: client.runtime.getManifest().version });
 		}
 	}
+
```
Comment (1zizi1) [inline src/Storage.ts:94]: In this file I get this error:
```js
index.js:183 
        
       Uncaught (in promise) TypeError: Error in invocation of storage.get(optional [string|array|object] keys, function callback): No matching signature.
    at Function.getIncludeRetweeters (index.js:183:27)
    at I.<anonymous> (index.js:498:39)
    at Generator.next (<anonymous>)
    at index.js:30:32
    at new Promise (<anonymous>)
    at t (index.js:4:10)
    at I.createCheckbox (index.js:491:11)
    at I.<anonymous> (index.js:718:25)
    at Generator.next (<anonymous>)
    at n (index.js:7:10)

```

**Review by 1zizi1 — CHANGES_REQUESTED** [review]

File: src/Storage.ts (hunk 1 above)
Comment (1zizi1) [reply src/Storage.ts:94]: This is fixed by upgrading the manifest version to 3. This API is only supported on V3.

See #90 

https://github.com/dmstern/likers-blocker/pull/90/commits/b4b72168a69261b0b429ff90e82ea50fb79a34dd

Comment (dmstern) [reply src/Storage.ts:94]: Great, thank you for your contribution! :)
✔️ Merged.

**Review by alphaleadership — COMMENTED** [review]
goof

**Comment by dmstern** [comment]
// closed in favor of #111

**Pull request closed without merging**